prime-angular-frontend/src/app/shared/services/banner-resource.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
+
+import { BannerResourceService } from './banner-resource.service';
+
+describe('BannerResourceService', () => {
+  let service: BannerResourceService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({});
+    service = TestBed.inject(BannerResourceService);
+  });
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
```

Comment:
Similar failed test to `BannerMaintenanceComponent`. Might require DI configuration. as well as `BannerWrapperComponent`